docs/core/tools/dotnet-run.md

```diff
@@ -74,213 +45,85 @@ To run the application, the `dotnet run` command resolves the dependencies of th
-
-`--runtime <RUNTIME_IDENTIFIER>`
-
-Specifies the target runtime to restore packages for. For a list of Runtime Identifiers (RIDs), see the [RID catalog](../rid-catalog.md).
-
-`-v|--verbosity <LEVEL>`
-
-Sets the verbosity level of the command. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`.
-
-# [.NET Core 2.0](#tab/netcore20)
-
-`--`
-
-Delimits arguments to `dotnet run` from arguments for the application being run. All arguments after this delimiter are passed to the application run.
-
-`-c|--configuration {Debug|Release}`
-
-Defines the build configuration. The default for most projects value is `Debug`.
-
-`-f|--framework <FRAMEWORK>`
-
-Builds and runs the app using the specified [framework](../../standard/frameworks.md). The framework must be specified in the project file.
-
-`--force`
-
-Forces all dependencies to be resolved even if the last restore was successful. Specifying this flag is the same as deleting the *project.assets.json* file.
-
-`-h|--help`
-
-Prints out a short help for the command.
-
-`--launch-profile <NAME>`
-
-The name of the launch profile (if any) to use when launching the application. Launch profiles are defined in the *launchSettings.json* file and are typically called `Development`,
-`Staging`, and `Production`. For more information, see [Working with multiple environments](/aspnet/core/fundamentals/environments).
-
-`--no-build`
-
-Doesn't build the project before running. It also implicit sets the `--no-restore` flag.
-
-`--no-dependencies`
-
-When restoring a project with project-to-project (P2P) references, restores the root project and not the references.
-
-`--no-launch-profile`
-
-Doesn't try to use *launchSettings.json* to configure the application.
-
-`--no-restore`
-
-Doesn't execute an implicit restore when running the command.
-
-`-p|--project <PATH>`
-
-Specifies the path of the project file to run (folder name or full path). If not specified, it defaults to the current directory.
-
-`--runtime <RUNTIME_IDENTIFIER>`
-
-Specifies the target runtime to restore packages for. For a list of Runtime Identifiers (RIDs), see the [RID catalog](../rid-catalog.md).
-
-# [.NET Core 1.x](#tab/netcore1x)
-
-`--`
-
-Delimits arguments to `dotnet run` from arguments for the application being run. All arguments after this delimiter are passed to the application run.
-
-`-c|--configuration {Debug|Release}`
-
-Defines the build configuration. The default value for most projects is `Debug`.
-
-`-f|--framework <FRAMEWORK>`
-
-Builds and runs the app using the specified [framework](../../standard/frameworks.md). The framework must be specified in the project file.
-
-`-h|--help`
-
-Prints out a short help for the command.
-
-`-p|--project <PATH/PROJECT.csproj>`
-
-Specifies the path and name of the project file. (See the NOTE.) If not specified, it defaults to the current directory.
-
-> [!NOTE]
```

Comment:
this is related to the e-mail I just sent. How far back do we want to cover the SDK versions in the docs?